packages/babel-parser/src/tokenizer/index.js

```diff
@@ -1250,9 +1262,13 @@ export default class Tokenizer extends LocationParser {
 
   // Used to read character escape sequences ('\x', '\u').
 
-  readHexChar(len: number, throwOnInvalid: boolean): number | null {
+  readHexChar(
```

Comment:
`readHexChar` is only used for `\x` and `\u`, so we don't have to expose `allowNumSeparator` to its caller.

Reply:
:thinking: Good catch. So, now it is exposed to `readInt`'s callers and changed to `false` when called from `readHexChar`.